test/JDBC/init.sh

```diff
@@ -30,6 +30,7 @@ GRANT ALL ON SCHEMA sys to jdbc_user;
 ALTER USER jdbc_user CREATEDB;
 \c jdbc_testdb
 ALTER SYSTEM SET babelfishpg_tsql.database_name = 'jdbc_testdb';
+ALTER SYSTEM SET babelfishpg_tsql.migration_mode = 'multi-db';
```

Comment:
Update the same here as well: https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_4_X_DEV/contrib/babelfishpg_tds/test/TDSNode.pm#L68

Reply:
Change in migration mode for TAP tests can be done separately, it would require resolving failures.